style.css

```diff
@@ -0,0 +1,65 @@
+@media (min-width: 375px), (min-width: 768px) {
+  .Intro {
+    background-image: url("background.svg");
+  }
+
```

Comment:
- [ ] Great work fixing the broken link of the background images as suggested by the previous reviewer. Currently, your background images are repeating so I would suggest you add a `background-repeat` and set it to `no-repeat` such that this issue can be fixed. See below for more details:
 - Also please make sure to position you background images properly to match that of the `Figma` design template so you design looks more better to create a good user experience.
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="375" alt="Screenshot 2023-06-02 at 11 44 24" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/107192076/fd314b4a-c1c0-4778-8aa6-3445c534daf3">
</td>
<td>
<img width="300" alt="Screenshot 2023-06-02 at 11 43 11" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/107192076/4b33cdd8-dc1f-4f35-b4ef-af05d6cf548d">
</td>
</tr>
</table>